java-client/example-utilities/src/main/java/io/deephaven/client/examples/RawTicket.java

```diff
@@ -0,0 +1,17 @@
+package io.deephaven.client.examples;
+
+import io.deephaven.client.impl.HasTicketId;
+import io.deephaven.client.impl.TicketId;
+import picocli.CommandLine.Option;
+
+import java.nio.charset.StandardCharsets;
+
+public class RawTicket implements HasTicketId {
+    @Option(names = {"--ticket"}, required = true, description = "The raw ticket.")
+    String ticket;
+
+    @Override
+    public TicketId ticketId() {
+        return new TicketId(ticket.getBytes(StandardCharsets.UTF_8));
```

Comment:
to confirm, only bytes-as-utf8 from the command line is allowed to be passed in this way? if I had to guess, i'd suggest this should ask for ASCII instead, or use base64 to be clear that tickets are binary

Reply:
I'm not sure the best way to do this - I'm not necessarily trying to support arbitrary bytes via CLI arguments, but just make it so that "tickets that look like strings" can be passed this way. The picocli library will have some way of creating `String ticket`, maybe with unicode support (maybe it depends on the terminal).

Reply:
Probably should ask the terminal what charset it uses (mine isn't defaulting to utf8 on at least one box), or assume ascii so we can fail in the general case?